lightning/src/chain/channelmonitor.rs

```diff
@@ -1382,17 +1382,24 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitor<Signer> {
 	/// Loads the funding txo and outputs to watch into the given `chain::Filter` by repeatedly
 	/// calling `chain::Filter::register_output` and `chain::Filter::register_tx` until all outputs
 	/// have been registered.
-	pub fn load_outputs_to_watch<F: Deref>(&self, filter: &F) where F::Target: chain::Filter {
+	pub fn load_outputs_to_watch<F: Deref, L: Deref>(&self, filter: &F, logger: &L) 
+	where 
+		F::Target: chain::Filter, L::Target: Logger,
+	{
 		let lock = self.inner.lock().unwrap();
 		filter.register_tx(&lock.get_funding_txo().0.txid, &lock.get_funding_txo().1);
+		log_trace!(logger, 
+			"Registering funding outpoint {}", &lock.get_funding_txo().0);
```

Comment:
@TheBlueMatt Why do we want to log the outpoint here when we are registering a txid and script pubkey? The outpoint corresponding to the funding transaction output will be registered and logged below, IIUC.